src/client/index.js

```diff
@@ -68,7 +71,12 @@ class Hammerhead {
             event:    eventUtils,
             position: positionUtils,
             style:    styleUtils,
-            types:    typeUtils
+            types:    typeUtils,
+            service:  {
+                isJQueryObj: isJQueryObj,
+                extend:      extend,
+                trim:        trim
+            }
```

Comment:
I'm agree with this. We have separated modules for this utils, so we should add the methods to the utils root.